### PR TITLE
moar updates to send messages to the right people

### PR DESCRIPTION
**Pull request by angaither**
Update user query to get last 30 day users.
Remove some test code.

### CODE REVIEW

File: app/Console/Commands/toto.php [hunk 1]
```diff
@@ -32,19 +32,20 @@ class Toto extends Command
...
-          'json' => [
-            'user' => $user,
-            'message' => $message,
-          ]
-      ]);
-
+      foreach ($users as $user) {
+        $message = Message::getMessage($user);
+        $client = new \GuzzleHttp\Client([
+          'base_uri' => 'http://localhost:3001',
```
Comment (sbsmith86) [inline app/Console/Commands/toto.php:40]: should this uri be set in the .env?

File: index.js [hunk 2]
```diff
@@ -54,7 +53,6 @@ slack.on('message', function(message) {
   // }
 
   var channel = slack.getChannelGroupOrDMByID(message.channel);
-  var response = message.text.split('').reverse().join('');
   channel.send('woof');
```
Comment (sbsmith86) [inline index.js:56]: :dog:

File: app/Console/Commands/toto.php (hunk 1 above)
Comment (angaither) [inline app/Console/Commands/toto.php:40]: hm, maybe. but it's the same for all local things...

Comment (sbsmith86) [inline app/Console/Commands/toto.php:40]: ah just wondering about when it gets deployed but probably not important

**Comment by sbsmith86** [comment]
:100:

File: app/Console/Commands/toto.php (hunk 1 above)
Comment (angaither) [inline app/Console/Commands/toto.php:40]: yeah, totally should be moved if we make this a real live thing. I'll open an issue for it.

**Pull request merged**